debian/rules

```diff
@@ -166,8 +168,9 @@ override_dh_install:
 		>debian/tmp/var/lib/delphix-appliance/platform
 	rm debian/tmp/var/lib/delphix-appliance/platform.in
 
-	./scripts/download-signature-key.sh upgrade 5.3 \
-		>debian/tmp/var/lib/delphix-appliance/key-public.pem.upgrade.5.3
+	for type in $(SIGN_KEY_TYPES) ; do \
+		./scripts/download-signature-key.sh $$type "$(DELPHIX_SIGNATURE_VERSION)" \
```

Comment:
When the second parameter to `download-signature-key.sh` is empty, the script will fail.. this causes our PR checks to fail, and likely will also prevent us from using `git-linux-pkg` to build and deploy.

We should ensure building the package still works (i.e. doesn't fail) when building without the `DELPHIX_SIGNATURE_VERSION` variable being available. E.g. I'd be OK augmenting the `download-signature-key.sh` script to return "success" when an empty version is passed in, and simply have it result in nothing being done (like we do when `DELPHIX_SIGNATURE_URL` and `DELPHIX_SIGNATURE_TOKEN` are not specified).

Reply:
Yeah earlier had a default for `DELPHIX_SIGNATURE_VERSION` and will make changes to follow similar to when `DELPHIX_SIGNATURE_URL` not specified.